data/FreeBSD-family.yaml

```diff
@@ -5,3 +5,4 @@ letsencrypt::cron_owner_group: 'wheel'
 letsencrypt::plugin::dns_rfc2136::package_name: 'py38-certbot-dns-rfc2136'
 letsencrypt::plugin::dns_route53::package_name: 'py38-certbot-dns-route53'
 letsencrypt::plugin::dns_cloudflare::package_name: 'py38-certbot-dns-cloudflare'
+letsencrypt::plugin::dns_gandi::package_name: 'certbot-plugin-gandi'
```

Comment:
There are currently no FreeBSD port for this :-/

The name would probably be `py38-certbot-dns-gandi` if there was, and in fact `py39-certbot-dns-gandi` because the default version of Python changed recently on FreeBSD.

Reply:
I opened #296 to adjust the FreeBSD version.

Reply:
Do you think it's a good idea to install the package with pip? I'm not sure on how to install them on freebsd and centos. I will ask help to a colleague

Reply:
As a FreeBSD user, I would prefer to use a package rather installing using pip.  Anybody can submit ports to FreeBSD, and they are ultimately available as packages.  My guess is that if the port does not exist, nobody needed it so much they did a port for it, and maybe it is not worth it?

I would personally not set a `package_name` for FreeBSD in Hiera and make `$letsencrypt::plugin::dns_gandi::package_name` mandatory so that it fails hard if someone attempt to use it on FreeBSD.